ui/packages/catalog-ui-search/package.json

```diff
@@ -25,6 +25,7 @@
     "node": ">=0.10.5"
   },
   "devDependencies": {
+    "@types/styled-components": "4.0.3",
```

Comment:
Styled doesn't contain it's type definitions in the base dependency?

Reply:
No, a lot of libraries move their definitions to @typed.  They did with v4 as well.  You'll notice this isn't the latest version though: https://github.com/styled-components/styled-components-website/issues/447 

If we have bandwidth it'd be nice to help them fix up their types so we can use the latest type definitions they have.  But for now it'll mess up theming.